nni/algorithms/compression/v2/pytorch/pruning/basic_pruner.py

```diff
@@ -165,27 +165,47 @@ class LevelPruner(BasicPruner):
         the non-zero value in tensor which create chance for better hardware acceleration.
 
         Note: If length of given balance_gran smaller than length of pruning tensor shape, it will be made up
-              in right align(such as example 1).
+        in right align(such as example 1).
+
+        Example 1::
+
+            operation: Linear
+            pruning tensor: weight
+            pruning tensor shape: [32, 32]
+            sparsity: 50%
+            balance_gran: [4]
+            pruning result: Weight tensor whose shape is [32, 32] will be split into 256 [1, 4] sub blocks.
+                            Each sub block will be pruned 2 values.
+
+        Example 2::
+
+            operation: Linear
+            pruning tensor: weight
+            pruning tensor shape: [64, 64]
+            sparsity: 25%
+            balance_gran: [32, 32]
+            pruning result: Weight tensor whose shape is [64, 64] will be split into 4 [32, 32] sub blocks.
+                            Each sub block will be pruned 256 values.
+    block_sparse_size : Union[int, List[int], None]
+        (experimental) By default, block_sparse_size is None, which means element-wise pruning with block size [1, 1, ..., 1].
+        If block_sparse_size is set, pruner will treat the block as a whole to prune.
```

Comment:
what if user sets both block_sparse_size and balance_gran?

Reply:
> > What will happen when block size smaller than balance granularity?
> 
> Balance granularity treat block as a pruning unit.
> 
> example:
> 
> ```
> balance_gran = [1, 2]
> block_size = [2, 2]
> masks = 
> [[0, 0, 1, 1, 1, 1, 0, 0],
>  [0, 0, 1, 1, 1, 1, 0, 0],
>  [0, 0, 1, 1, 0, 0, 1, 1],
>  [0, 0, 1, 1, 0, 0, 1, 1]]
> ```